lib/qs/payload.rb

```diff
@@ -1,17 +0,0 @@
-require 'oj'
-
-module Qs
-
-  module Payload
-
-    def self.encode(payload)
-      Oj.dump(payload, :mode => :strict)
```

Comment:
@jcredding since we have the top-level `Qs.serialize` and `Qs.deserialize` methods, we could just switch to using them here and not have change the client or daemon implementations.  I'd probably rename these payload methods to serialize/deserialize to match.

That would turn this into a glorified demeter method module, I know.  I like the "payload" abstraction though.  I'd also be in favor of just using `Qs.serialize` in place of `Payload.encode` as well.